### DISCUSSION TITLE
forward nodes, remote sites, vpn tunnels?

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#6000] [Unsupported Versions] [answered]
Thinking of deploying forward nodes in remote sites, I have VPN tunnels that all talk to each other, can I send logs to my manager+search (at my location) from each sit?. I have been digging haven't seen anyone talk about this yet. Is this ok because  I have limited resources to have a manager+search at each location. I can just so-allow all the tunnels that I want...anyone have this setup? and add to my grid via soremote, manager IP etc?

**Comment by dougburks** [comment, marked as answer]
Yes, you should be able to use VPN:
#1139

**Reply by Sahale** [reply]
Just to be sure: IPSec works too? Thanks!

**Reply by dougburks** [reply]
Currently, Setup should correctly detect and properly handle tun*, wg*, and vpn* interfaces:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/setup/so-functions#L922

If you have further questions, please start a new discussion.
